site/content/docs/manifest/pipeline.en.md

```diff
@@ -2,31 +2,31 @@ List of all available properties for a Copilot pipeline manifest. To learn more
 
 ???+ note "Sample manifest for a pipeline triggered from a GitHub repo"
 
-```yaml
-    name: pipeline-sample-app-frontend
-    version: 1
-
-    source:
-      provider: GitHub
-      properties:
-        branch: main
-        repository: https://github.com/<user>/sample-app-frontend
-        # Optional: specify the name of an existing CodeStar Connections connection.
-        connection_name: a-connection
-
-    build:
-      image: aws/codebuild/amazonlinux2-x86_64-standard:3.0
-
-    stages:
-        -
-          name: test
-          test_commands:
-            - make test
-            - echo "woo! Tests passed"
-        -
-          name: prod
-          requires_approval: true
-```
+    ```yaml
+        name: pipeline-sample-app-frontend
```

Comment:
We need to change this to our new format...
```suggestion
        name: frontend
```

Reply:
Or we can leave it to the multiple pipeline doc changes.

Reply:
Done!